tools/src/main/java/org/apache/rocketmq/tools/command/MQAdminStartup.java

```diff
@@ -243,4 +245,25 @@ private static SubCommand findSubCommand(final String name) {
     public static void initCommand(SubCommand command) {
         subCommandList.add(command);
     }
+
+    public static RPCHook getAclRPCHook() {
+        String fileHome = System.getProperty(MixAll.ROCKETMQ_HOME_PROPERTY, System.getenv(MixAll.ROCKETMQ_HOME_ENV));
+        String fileName = "/conf/tools.yml";
+        JSONObject yamlDataObject = AclUtils.getYamlDataObject(fileHome + fileName ,
```

Comment:
We must pay more attention to the failure of the throwable exception when using fastjson, please check and catch throwable if that thing exists.